de.dlr.sc.virsat.external.lib.commons.cli/plugin.xml

```diff
@@ -8,6 +8,9 @@
      <clArgs
            command="superUser">
      </clArgs>
+     <clArgs
+           command="demo">

```

Comment:
Great Idea with the command line option. Still i believe it is not a "demo" mode the software will be set to. It is more like a "ignoreBetaConcepts" mode.

Thus the command line option should be called "-ignoreBetaConcepts"